src/components/settings/import-and-export/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { SettingsContainer } from '@/components/ui/styled.ts'
+import { Label } from '@/components/ui/label'
+import { Button } from '@/components/ui/button.tsx'
+import { Import, Share } from 'lucide-react'
+import { TextArea } from '@radix-ui/themes'
+import '@radix-ui/themes/styles.css'
+import { useStore } from '@/store'
+import { useState } from 'react'
+import { t } from 'i18next'
+
+export const ImportAndExportSettings = () => {
+  const cls = 'col-span-2 grid grid-cols-subgrid items-center'
+  const [style, setStyle] = useStore((store) => store.styles)
+  const [showData, setShowData] = useState('')
+  return (
+    <SettingsContainer>
+      <div className={cls}>
+        <Label>{t('export data')}</Label>
+        <Button
+          size={'icon'}
+          onClick={() => {
+            setShowData(btoa(JSON.stringify(style)))
```

Comment:
Wouldn't base64 be too long? Maybe using a plain JSON string directly would be better.